cardano-testnet/src/Testnet/Components/Query.hs

```diff
@@ -301,7 +302,12 @@ checkDRepState sbe configurationFile socketPath execConfig f = withFrozenCallSta
                   [ "checkDRepState: foldEpochState returned Nothing: "
                   , "This is probably an error related to foldEpochState." ]
       H.failure
-    Right (_, Just val) ->
+    Right (ConditionNotMet, Just _) -> do
```

Comment:
Just out of curiosity, has this happened for you or is this just a safeguard?

Reply:
Safeguard